src/Sentry/SentryOptions.cs

```diff
@@ -401,6 +401,31 @@ public SentryOptions()
                     "Serilog",
                     "Giraffe.",
                     "NLog",
+                    "Npgsql",
+                    "RabbitMQ",
+                    "Hangfire",
+                    "IdentityServer4",
+                    "AWSSDK",
+                    "Polly",
+                    "Swashbuckle",
+                    "Fluentassertions",
```

Comment:
Why would someone use Fluentassertions in an application? Should it be FluentValidation?

Reply:
yes